ibm/mas_devops/roles/ocp_setup_mas_deps/templates/sbo/subscription.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
I think this is the result of a bad merge and this file needs to be deleted.  [subscription-stable.yml](https://github.com/ibm-mas/ansible-devops/blob/ocp-role-merge-packet/ibm/mas_devops/roles/ocp_setup_mas_deps/templates/sbo/subscription-stable.yml) and [subscription-preview.yml](https://github.com/ibm-mas/ansible-devops/blob/ocp-role-merge-packet/ibm/mas_devops/roles/ocp_setup_mas_deps/templates/sbo/subscription-preview.yml) have replaced this file

Reply:
correct, deleted this file